.ci_support/linux_64_python3.6.____73_pypy.yaml

```diff
@@ -0,0 +1,22 @@
+c_compiler:
+- gcc
+c_compiler_version:
+- '7'
+channel_sources:
+- conda-forge,defaults
+channel_targets:
+- conda-forge main
+cxx_compiler:
+- gxx
+cxx_compiler_version:
+- '7'
+docker_image:
+- condaforge/linux-anvil-comp7
+pin_run_as_build:
+  python:
+    min_pin: x.x
```

Comment:
Should this really be `x.x`?

Reply:
I think this is autoreplaced at some point by the version on line 20.

Anyway, these files should not be edited by humans IMO. At least, in the past I never changed anything outside the recipe dir and let the CI do the rest.

Reply:
Ok